src/main/java/net/trollyloki/advancementbingo/BingoBoard.java

```diff
@@ -240,6 +248,42 @@ public boolean complete(@NotNull NamespacedKey advancementKey) {
         return true;
     }
 
```

Comment:
Is there a way to reduce the amount of code that's repeated between these new getProgress methods and the old isComplete methods?

Reply:
sure - the isComplete methods now use the getProgress methods so it's shorter

originally I was going to leave it unchanged because the old methods were more efficient but it probably didn't matter